core/global/display_manager.gd

```diff
@@ -20,9 +20,9 @@ signal brightness_changed
 const backlight_path := "/sys/class/backlight"
 const steamos_write_bin := "/usr/bin/steamos-polkit-helpers/steamos-priv-write"
 
+var logger := Log.get_logger("DisplayManager", Log.LEVEL.DEBUG)
```

Comment:
Not a deal breaker, but we should start looking out for these to be INFO in main PR's